wrapper-bench-tps.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -e
+exec > wrapper-bench-tps.log
+exec 2>&1
+
+tdsDir="$(pwd)"
+here=$(dirname "$0")
+netDir=$1
+clientId=$2
+txCount=$3
+
+cd $netDir
```

Comment:
IMO, avoid the `cd` and just path to each net/ command explicitly.   Net can find itself

Reply:
Ah yeah, good point!